src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -933,11 +933,27 @@ gl_sframe drawing_classifier::predict_topk(gl_sframe data,
 
 variant_map_type drawing_classifier::evaluate(gl_sframe data,
                                               std::string metric) {
-  gl_sarray predictions = predict(data, "probability_vector");
 
-  return evaluation::compute_classifier_metrics(
-      data, read_state<flex_string>("target"), metric, predictions,
+  gl_sarray predictions_prob = predict(data, "probability_vector");
+
+  flex_list class_labels = read_state<flex_list>("classes");
+    auto max_prob_label = [=](const flexible_type& ft) {
+      const flex_vec& prob_vec = ft.get<flex_vec>();
+      auto max_it = std::max_element(prob_vec.begin(), prob_vec.end());
```

Comment:
Is the use of `auto` here increasing readability?

Reply:
Yes. And also follow other code's style.